### ISSUE TITLE
No Dark Mode on Homescreen

### ISSUE THREAD

**Issue by MithNo1** [edrlab/thorium-reader#1915]
I'm not sure if this not added? Or am I missing something? The dark mode works well inside when reading but in home screen where books are well organized there is no dark mode! Can you please add that? Or am I missing something? On Windows PC and latest version of thorium reader

**Comment by danielweck** [comment]
This feature is currently not implemented. TODO.

**Comment by danielweck** [comment]
Implemented in Thorium 3
